generators/client/templates/angular/webpack/webpack.common.js.ejs

```diff
@@ -59,7 +59,8 @@ module.exports = (options) => ({
                 options: {
                     digest: 'hex',
                     hash: 'sha512',
-                    name: 'content/[hash].[ext]'
+                    name: 'content/[hash].[ext]',
+                    esModule: false
```

Comment:
I think we should add a comment listing out the reason for this change and revisit this after the next `html-loader` release?

Reply:
Can you please add the requested comment @opt-nc-dev?

Reply:
I added the comment